README.md

```diff
@@ -358,3 +358,56 @@ Please check out common DSC Community [contributing guidelines](https://dsccommu
 
 - [CMAssetIntelligencePoint_Absent](Source\Examples\Resources\CMAssetIntelligencePoint\CMAssetIntelligencePoint_Absent.ps1)
 - [CMAssetIntelligencePoint_Present](Source\Examples\Resources\CMAssetIntelligencePoint\CMAssetIntelligencePoint_Present.ps1)
+
+### CMDistributionPoint
+
+- **[String] SiteCode** _(Key)_: Specifies the Site Code for the Configuration
+  Manager site.
+- **[String] SiteServerName** _(Key)_: Specifies the SiteServer to install the role.
+- **[String] Description** _(Write)_: Specifies a description for the
+  distribution point.
+- **[UInt32] MinimumFreeSpaceMB** _(Write)_: Specifies the amount of free space
```

Comment:
As per our discussion, it doesn't make since to break it out at this point.  Just to add, if more people request it may be a feature request later on.